docs/reference/upgrade/rolling_upgrade.asciidoc

```diff
@@ -120,6 +120,8 @@ GET _cat/health
 
 Wait for the `status` column to switch from `yellow` to `green`. Once the
```

Comment:
I think we could drop any mention of the starting state here. It's true that it might start as `red`, and also that it might go straight from `red` to `green` without ever being `yellow`, but none of that is especially important here.

```suggestion
Wait for the `status` column to switch to `green`. Once the
```

Reply:
TBC I think we should do this instead of adding the extra sentence suggested in this PR.

Reply:
I think that's a great idea. Implemented with 366586b.